docs/ops/infrastructure/Loop_5.md

```diff
@@ -0,0 +1,183 @@
+
+        * *external_port_id*
+            * **Description**: *external_port_id* is a port ID of the `Loop` operation.
+            * **Range of values**: indexes of the *Loop* outputs
+            * **Type**: `int`
+            * **Default value**: None
+            * **Required**: *yes*
+
+        * *internal_operation_id*
+
+            * **Description**: *internal_operation_id* is a `Parameter` or `Result` operation ID inside the `body` network to map to.
+            * **Range of values**: IDs of the `Parameter` operations inside in the *Loop* operation
+            * **Type**: `int`
+            * **Default value**: None
+            * **Required**: *yes*
+
+        * *axis*
+
+            * **Description**: *axis* is an axis to concatenate the body `Result` output across all iterations. Can be specified for `output` entry only.
+            * **Range of values**: an integer
+            * **Type**: `int`
+            * **Default value**: None
+            * **Required**: *no*
+
+* **Back edges**:
+
+    *back_edges* is a set of rules to transfer tensor values from `body` outputs at one iteration to `body` parameters at the next iteration. Back edge connects some `Result` operation in the `body` to `Parameter` operation in the same `body`.
+
+    * **Back edge attributes**:
+
+        * *from-operation*
+
+            * **Description**: *from-operation* is a `Result` operation ID inside the `body` network.
+            * **Range of values**: IDs of the `Result` operations inside the *Loop*
+            * **Type**: `int`
+            * **Default value**: None
+            * **Required**: *yes*
+
+        * *to-operation*
+
+            * **Description**: *to-operation* is a `Parameter` operation ID inside the `body` network to end mapping.
+            * **Range of values**: IDs of the `Parameter` operations inside the *Loop*
+            * **Type**: `int`
+            * **Default value**: None
+            * **Required**: *yes*
+
+**Loop Inputs**
+
+* **Trip count**: A scalar tensor of `int64` type specifying maximum number of iterations. *Required*.
+
+* **Termination condition**: A scalar tensor of `boolean` type specifying whether to execute the first iteration or not. *Required*.
+
+* **Multiple other inputs**: tensors of different types and shapes. *Optional*.
+
+**Loop Outputs**
+
+* **Multiple outputs**: Results of execution of the `body`. Tensors of any type and shape.
+
+
+**Body Inputs**
+
+* **Current iteration**: A scalar tensor of `int64` type specifying the current iteration number. *Required*.
+
+* **Multiple other inputs**: tensors of different types and shapes. *Optional*.
+
+
+**Body Outputs**
+
+* **Termination condition**: A scalar tensor of `boolean` type specifying whether to execute the next iteration or not.
+
+* **Multiple outputs**: Results of execution of the `body`. Tensors of any type and shape.
+
+
+**Examples**
+
+*Example 1: a typical Loop structure*
+```xml
+<operation type="Loop" ... >
```

Comment:
Do we already change tag name from `layer` to `operation`?

Reply:
The currect MO version generates `layer`. Will be fixed.

Reply:
Renamed